### ISSUE TITLE
Erroneous data on CHEMBL98511

### ISSUE THREAD

**Issue by Evert-Homan** [chembl/GLaDOS#1308]
CHEMBL98511 has a KI value of 5 nM pKi 8.3) in ChEMBL_31 for HTR6. Based on the literature reference CHEMBL1145819 it appears that this is based on the sentence:

'The corresponding racemic octahydropyrido[1,2-a]pyrazine (Ki = 5 nM) was shown to bind with somewhat reduced affinity.'

It seems that the reported data hase been associated with octahydropyrido[1,2-a]pyrazine, which is a fragment of the actual structure referred to in the paper. Imo CHEMBL98511 should be removed from ChEMBL.

Best wishes/Evert

**Comment by scorbett-ebi** [comment]
CHEMBL1145819 (doc_id = 17066, doi 10.1021/jm030030n) is a review paper.  The original paper the quoted measurement is taken from is CHEMBL1135108 (doc_id = 4829, doi 10.1016/s0960-894x(02)00172-5).  The value in the original paper is a pKi of 8.3 which has been correctly assigned to compound CHEMBL29410 (molregno = 38499) under the document CHEMBL1135108 (activity_id = 241135).

The compound CHEMBL98511 (molregno = 155411) is only associated with one activity (doc_id = 17066, assay_id = 3664, activity_id = 82690) under document CHEMBL1145819.  It does appear that the compound is a fragment of the actual structure .

The simplest solution is to reassign the molregno for the single activity from CHEMBL98511 (molregno 155411) to CHEMBL431298 (molregno = 38445):

UPDATE chembl.activities
SET molregno = 38499, potential_duplicate = 1, original_activity_id = 241135
WHERE doc_id = 17066
AND assay_id = 3664
AND activity_id = 82690
AND molregno = 155411

Given that no other activities are associated with the structure CHEMBL98511 this could also be downgraded:

UPDATE chembl.molecule_dictionary
SET downgraded = 1, downgrade_reason = 'Incorrect structure'
WHERE molregno = 155411
AND chembl_id = 'CHEMBL98511'

**Comment by Evert-Homan** [comment]
I agree that the simplest solution is to reassign the molregno for the single activity from CHEMBL98511 (molregno 155411) to CHEMBL431298 (molregno = 38445), since CHEMBL98511 is merely a fragment of CHEMBL431298.

Best wishes/Evert

**Comment by Emma-Manners** [comment]
@scorbett-ebi 
Let's discuss tomorrow and I'll fix this one.
